docs/docs/topics/tcp-support.md

```diff
@@ -39,7 +39,7 @@ routes:
           - email:
               is: contractor@not-example.com
           - groups:
-              has: ["datascience@example.com"]
+              has: "datascience@example.com"
```

Comment:
I thought you said groups.has took an array, or is that only required when there are multiple values?